lib/model/unittest/CCountingModelTest.cc

```diff
@@ -23,6 +23,8 @@
 #include <string>
 #include <vector>
 
+#include "CModelTestFixtureBase.h"
```

Comment:
This should go after all the ML library includes but before the 3rd party includes, so after line 20 in this file (and similarly in the other 4 files).